### DISCUSSION TITLE
Trying to change my radio

### DISCUSSION THREAD

**Discussion by mihirat33** [cedricp/ddt4all#939] [Q&A] [answered]
Trying to change my radio on Renault scenic 3, when I attempting to connect with audio ecu I receiving  a msg :
Found ECU [Audio] (no relevant ECU file found) :DIAGVERSION [164] SUPPLIER [BT2] SOFT [1003] VERSION [0000]
Is a DB issue and how to solve.

**Comment by AndyIreland** [comment]
have you re-wired your ELM327 or Rewired a OBD2 fly lead? 

you need to wire it up like this so you can read multimedia through the OBD2 socket
https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/

**Comment by mihirat33** [comment]
Yes I re-wired my ELM327. Before that audio ECU was not in list.

**Comment by AndyIreland** [comment]
ah right , what radio have you got in your scenic 3 - is it a rlink1 rlink2  touchscreen LCD display? or just basic radio?

**Comment by mihirat33** [comment]
This one 
![image](https://github.com/cedricp/ddt4all/assets/150245158/dd55a23a-12f0-4783-8d3e-5f83ba95c974)
trying to change with this
![download](https://github.com/cedricp/ddt4all/assets/150245158/93c666ee-7870-4a76-9830-d17e514a8d9d)

**Comment by KarelSvo** [comment]
R1_08_V1.3_20090612T085711.xml

**Comment by AndyIreland** [comment]
if you want the xml file I have uploaded it to here for you. - I wonder if you will have to fill out the VIN details as well if its come out of another car? 
https://we.tl/t-8QkL0TbcgP

**Reply by elusivestudio** [reply]
> if you want the xml file I have uploaded it to here for you. - I wonder if you will have to fill out the VIN details as well if its come out of another car? https://we.tl/t-8QkL0TbcgP

Hi Andy,
This link is expired, can you please share it again?

Also I am stuck in unconfigured radio / EOL settings error screen.
Any step by step guide you can provide?

Thank you,
Sem

**Reply by AndyIreland** [reply]
Hi the xml file do you need is it? - What version is your r-link radio. Is it rlink1 or rlink2?

**Comment by KarelSvo** [comment]
The file is not missing, it just needs to be selected manually because the Autoident data is incomplete.

**Comment by mihirat33** [comment]
Little update. Trying to change R1_08_V1.3 with R2_08_V3_81. All tests ok but nothing on the LCD (the type of radio is chaged in lcd ecu) and no sound on speacers. During the test speacers react normaly ( sound yes) but traing to set the tuner on no sound. Also I can't write Accoustic configuration, I am taking parameters from [here](https://gitlab.com/py_ren/pyren/-/blob/master/pyren/macro/megane3_fluence/radiosat_classic_bluetooth_accoustic.cmd)

**Comment by mihirat33** [comment]
I have managet to connect the radio wth central display, only think that left are the accoustic parameters. I've made diferent atempts using Pyren or ddt4all diferent com speed using macro or ddt4all but no success till naw. Please help.

**Reply by AndyIreland** [reply]
did you fill out your VIN number and CRC details?

**Reply by mihirat33** [reply]
Yes. The error msg is incomplete configuration.

**Reply by AndyIreland** [reply]
you could do with dump off similar car with the same radio and restore it in PyREN and just replace with your VIN and CRC number (if thats possible) just so it fills out the missing parameters. Might work.

**Comment by KarelSvo** [comment]
log files

**Reply by mihirat33** [reply]
there is
[screens.txt](https://github.com/cedricp/ddt4all/files/13429468/screens.txt)

**Comment by KarelSvo** [comment]
files

**Reply by mihirat33** [reply]
thos ones?
[ecu_ddt.txt](https://github.com/cedricp/ddt4all/files/13429684/ecu_ddt.txt)
[elm_ddt.txt](https://github.com/cedricp/ddt4all/files/13429686/elm_ddt.txt)

**Comment by mihirat33** [comment]
It seems that my OBD elm 1.3 is not capable to write or read such big clusters of data.

**Reply by Yamh631** [reply]
I had the same problem for write big data, here is my thread about it: #921

**Comment by mihirat33** [comment]
I managet to write accoustic parameters. Good or bad i dont know but there are. The audio don't start again only msg on screen "Connect diag un". Reading faults i recieve:

1  - DTC932813 (DF056) MICROPHONE                                         CO     PRESENT
2  - DTC934155 (DF030) COMP CONFIGURATION                                 1.DEF  PRESENT


ECU : 10728   (RT:0.0638 RR: 15.6)   R208_X95_X62_X61_0000_64_B
Screen : DE (STD_B) -> DTC934155 (DF030) COMP CONFIGURATION                                 1.DEF  PRESENT

        1.DEF:Configuration new or incomplete.



Additional data

PR001  MILEAGE SINCE LAST APPEARANCE OF FAULT                       185855 km

Press H for help or any key to exit

Any idea would be appreciated

**Reply by Yamh631** [reply]
Hello,

`1 - DTC932813 (DF056) MICROPHONE CO PRESENT`
--> Microphone CO = Circuit Open

Is your Scenic equipped with a microphone? 
![Capture d'écran 2023-11-26 235249](https://github.com/cedricp/ddt4all/assets/35656304/0503d17a-ed72-48d3-8538-6c8f656289ca)

----
`1.DEF:Configuration new or incomplete.`
The radio configuration is incomplete, you must have forgotten some parameters?

Did you enter the VIN / CRC correctly?
![Capture d'écran 2023-11-27 001320](https://github.com/cedricp/ddt4all/assets/35656304/05bd4ded-f601-4534-a4e8-c5e6d00a6e66)

If the car radio is new, it is blank and you must enter "all" parameters

----

His design radio is available in 2 different finishes:
RAD02: Classic Bluetooth
RAD03: 3D Sound by Arkamys (with Bluetooth too)

Here are the original advanced settings for my Mégane 3 Arkamys, I made a Macro Pyren if you want to try: [My Arkamys R2_08_v3_81.zip](https://github.com/cedricp/ddt4all/files/13468362/My.Arkamys.R2_08_v3_81.zip)

And my Dump (Full parameters) : [Dump - My Arkamys R2_08_v3_81.txt](https://github.com/cedricp/ddt4all/files/13468386/Dump.-.My.Arkamys.R2_08_v3_81.txt)

**Reply by mihirat33** [reply]
The microphon is missing but this can't be the reason for the radio to not start. I will try with your dump lately. For now the message  "Connect diag un" has disappeared but nothing else. This radio has a code but never prompted for it. I was wondering how to enter it.

**Reply by shrlnm** [reply]
@mihirat33 have you compared macros for your vin and the vin of donors car?

**Reply by mihirat33** [reply]
Unfortunately no. First thing I had done was to change the VIN. After that I have done reset of the configuration by mistake.

**Reply by mihirat33** [reply]
only thing that I saw is that car was Nissan.

**Reply by shrlnm** [reply]
@mihirat33 then I suggest you to find the vin of S3 with configurtion similar to yours but with R2_* module. It allows you to generate configuration for this radion in a context of you MTC parameters.
You may use pyren3/doc_macker with all_vin option

**Reply by mihirat33** [reply]
Doc_maker unfortunatly do not work on my computer. I still can compare the configuration of my old radio with this one. I think there was difference in Vehicle indentifre.
![image](https://github.com/cedricp/ddt4all/assets/150245158/452b55ce-6948-4136-b716-b144d0e74eeb)

**Reply by mihirat33** [reply]
![12](https://github.com/cedricp/ddt4all/assets/150245158/a999b840-cd67-4d65-925e-840d22d7e2f4)

**Reply by shrlnm** [reply]
> Doc_maker unfortunatly do not work on my computer.

Probably you don't have bvmextraction database

> I think there was difference in Vehicle indentifre.

It does not matter. You have to configure the new radio with full set of parameters according to yours MTC options

**Reply by mihirat33** [reply]
I do have bvmextraction database but starting doc_maker Python just blink and goes off. I was wondering do i have to install Python 3 and pyren3 version for doc_maker.

**Reply by shrlnm** [reply]
pyren3 is recommended. Old pyren on python2 is not supported anymore
When python script behave as you described, try to run it manually from the command line. It will show you error messages which suggest the root course of problem and may allow to solve it

**Reply by mihirat33** [reply]
Doc_maker terminate with wrong COM port MSG

CLIP DB : ..
DDT  DB : ..\DDT2000data_02_2021.zip
Loading language
Done
Opening ELM
ELM not connected or wrong COM port defined.

Available COM ports:

****************************************
*
*       ERRORS STATISTIC
*
* error_frame      =  0
* error_bufferfull =  0
* error_question   =  0
* error_nodata     =  0
* error_timeout    =  0
* error_rx         =  0
* error_can        =  0
*
*       RESPONSE TIME (Average)
*
* response_time    =  0.000
*
****************************************

**Reply by shrlnm** [reply]
Run is with parameters like this
./doc_maker.py --allvin XFB

**Reply by mihirat33** [reply]
It seems that doc_maker.py doesn't take the arguments in accaunt. I tryed with auto config_maker and this bat file-
"chcp 1251
set /p input="Ведите vin:"
pyren3\acf.py -ppp --demo --vin %input%
pause"
It is the same.

**Comment by KarelSvo** [comment]
[VF1JZ1U0646465286.htm.zip](https://github.com/cedricp/ddt4all/files/13492991/VF1JZ1U0646465286.htm.zip)

**Comment by mihirat33** [comment]
I have managed to create a script for this radio (R02) using MCT data for my vin and acf.py but the connect diag unit msg is still prsent and  no sound from the radio. Great thanks to @KarelSvo and @shrlnm for guidelines. This is the log it seems only phone acaustic are not changed 
[log.txt](https://github.com/cedricp/ddt4all/files/13587157/log.txt)

**Reply by shrlnm** [reply]
@mihirat33 What is shown by "ACI: Autoconfig info" for family 76 and 10? Probably the macro was made not correct as so there is "72:NR: General Programming Failure" in the log

**Reply by mihirat33** [reply]
for 76 is 76:283C20015R,283C20015R but i did not managet to create a script with this ACI (may be is not in the DB) and took ACI from anothr similar blok. There is not 10 family blok in this car according to MCT i will check later with PyRen.

**Comment by mihirat33** [comment]
ONLY thing that I recieve is: "Connect diag unit" on the display
All that i know for this radio: It is Blaupunkt part number and other info on the picture below.
ID info from PYREN:
ECU : 10728   (RT:0.0319 RR:  5.0)   R208_X95_X62_X61_0000_64_B
Screen : ID : Identifications
ID006  MPR REFERENCE                                      283C20015R
ID008  SUPPLIER                                           034
ID009  PROGRAM NUMBER                                     3807
ID010  VDIAG NUMBER                                       64
ID011  SOFTWARE VERSION                                   3807
ID012  CALIBRATION NUMBER                                 0000
ID013  ELECTRONIC VERSION                                 HARDW00000
ID014  VIN CODE                                           VF1JZ1U0646465286
ID005  DATE OF LAST AFTER SALES OPERATION                 100708
ID007  RADIO PROGRAMMING KEY                              H713q
![IMG_20240108_151347 1](https://github.com/cedricp/ddt4all/assets/150245158/2a48fea0-6e14-4151-81fb-a67dd3e4bda2)
If  someone could assist me to bring this radio lo life please help.

**Reply by shrlnm** [reply, marked as answer]
@mihirat33 
Let's try this macro

```
# VF1JZ1U0646465286

$addr = 13

#idTx = 0712 idRx = 0712

can500  # init can macro

delay 1

# open session
session 10C0

# configuration
2E2B01100708
2E2B0805
2E2B1100
2E2B1201
2E2B130F
2E2B1B00
2E2B1C20
2E2B2400
2E2B250101705AC7100401724E00029402950046E0A00066C0006666030B00005AA10000000810082E2838DD7A2607FF0032000476388A4567B5D409388A4A4267B536901B1B21C5413722543D6F900021C5FFC05C8331565B487754E5D3172A831D5B4862071F474CD3D53A251F37424CD36C00000000000001003400033E2002950000467EFFFC9246FCF81B9C3A9259BCFCF85B85C745E965734FF6DEBE238518E965795242391FBB3DBB1142B70D1FBB00440000000000050003380051EB00020D00027F8DCEFFFF00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
2E2B2B3E7752260001FFF8FDA401A5FE0031033E77510A0001FFF0FD9C035AFC00188100003E7751330001FFF5FD9D0251FC0018813E7763E20001FFE9FE1005FCFC0018810000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C00040000000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C00040000000375A40000001FF78FCD41424F0001E5A3ED340000001FFD5FDB90912F8001E563FD540000001FFFFFFAB00C7FF0023643FC840000001FFFFFF91018EFE001C0A3F5840000001FFF9FEB7028DFC002A093E1B40000001FFBFFC650889F000184500003F994000000100A2FF3600A3FF8065B3352240000001F52AF1243FD5C00028743AE940000001FED7F7BA0E9AF0005FE63D3C40000001FDCBFB003310E000189645314000000102FBFEF524BCC0002FB23FE840000001FFFFFFD10167FF001AE4000038E740000001FFFEFFBB0309FE0016C53E2340000001FF99FC700DD3F8002B863B8B40000001FA5AF88D5127C00014FC3FD640000001FFFFFFAD011DFF0025B600003F5240000001FFF9FEAB02C3FC0027473A8440000001FF39F6EB0914F00057F33FDA40000001FFFFFFB6011DFE0015293F7940000001008DFEF6008EFF00431E00000600060006000600200000000001000020000000000100002000000000010000200000000001000000000000009500000000009513100000000100001310000000010000
2E2B2C3E7752260001FFF8FDA401A5FE0031033E77510A0001FFF0FD9C035AFC00188100003E7751330001FFF5FD9D0251FC0018813E7763E20001FFE9FE1005FCFC0018810000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C00040000000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C00040000000375A40000001FF78FCD41424F0001E5A3ED340000001FFD5FDB90912F8001E563FD540000001FFFFFFAB00C7FF0023643FC840000001FFFFFF91018EFE001C0A3F5840000001FFF9FEB7028DFC002A093E1B40000001FFBFFC650889F000184500003F994000000100A2FF3600A3FF8065B3352240000001F52AF1243FD5C00028743AE940000001FED7F7BA0E9AF0005FE63D3C40000001FDCBFB003310E000189645314000000102FBFEF524BCC0002FB23FE840000001FFFFFFD10167FF001AE4000038E740000001FFFEFFBB0309FE0016C53E2340000001FF99FC700DD3F8002B863B8B40000001FA5AF88D5127C00014FC3FD640000001FFFFFFAD011DFF0025B600003F5240000001FFF9FEAB02C3FC0027473A8440000001FF39F6EB0914F00057F33FDA40000001FFFFFFB6011DFE0015293F7940000001008DFEF6008EFF00431E00000600060006000600200000000001000020000000000100002000000000010000200000000001000000000000009500000000009513100000000100001310000000010000
2E2B43FAE1C8AF967D644B321900
2E2B44FAB47D553C28190F080403

# VIN programming !!!check the command!!!
#2EF1905646314A5A315530363436343635323836
3B815646314A5A3155303634363436353238361A6E

# reset ecu or disconnect the battary!!!check the command!!!
1101

exit

```

**Reply by mihirat33** [reply]
Result
C:\Users\HP\Documents\pyren-pyren3\pyren-pyren3\main.py:430: SyntaxWarning: invalid escape sequence '\ '
  self.root.option_add('*Dialog.msg.font', 'Courier\ New 12')
openning file: ./macro/init.txt
Opening ELM
Checking port speed: 38400
Start COM speed:  38400
Changing baud rate to: 230400 OK
7A:7E0:ELM# # VF1JZ1U0646465286
7A:7E0:ELM#
7A:7E0:ELM# $addr = 13
13:712:ELM#
13:712:ELM# #idTx = 0712 idRx = 0712
13:712:ELM#
13:712:ELM# can500  # init can macro
13:712:ELM#
13:712:ELM# delay 1
13:712:ELM#
13:712:ELM# # open session
13:712:ELM# session 10C0
50 C0
# delay: 1.0
13:712:ELM#
13:712:ELM# # configuration
13:712:ELM# 2E2B01100708
6E 2B 01
# delay: 1.0
13:712:ELM# 2E2B0805
6E 2B 08
# delay: 1.0
13:712:ELM# 2E2B1100
6E 2B 11
# delay: 1.0
13:712:ELM# 2E2B1201
6E 2B 12
# delay: 1.0
13:712:ELM# 2E2B130F
6E 2B 13
# delay: 1.0
13:712:ELM# 2E2B1B00
6E 2B 1B
# delay: 1.0
13:712:ELM# 2E2B1C20
6E 2B 1C
# delay: 1.0
13:712:ELM# 2E2B2400
6E 2B 24
# delay: 1.0
13:712:ELM# 2E2B250101705AC7100401724E00029402950046E0A00066C0006666030B00005AA10000000810082E2838DD7A2607FF0032000476388A4567B5D409388A4A4267B536901B1B21C5413722543D6F900021C5FFC05C8331565B487754E5D3172A831D5B4862071F474CD3D53A251F37424CD36C00000000000001003400033E2002950000467EFFFC9246FCF81B9C3A9259BCFCF85B85C745E965734FF6DEBE238518E965795242391FBB3DBB1142B70D1FBB00440000000000050003380051EB00020D00027F8DCEFFFF00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
:72:NR: General Programming Failure
# delay: 1.0
13:712:ELM# 2E2B2B3E7752260001FFF8FDA401A5FE0031033E77510A0001FFF0FD9C035AFC00188100003E7751330001FFF5FD9D0251FC0018813E7763E20001FFE9FE1005FCFC0018810000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C00040000000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C00040000000375A40000001FF78FCD41424F0001E5A3ED340000001FFD5FDB90912F8001E563FD540000001FFFFFFAB00C7FF0023643FC840000001FFFFFF91018EFE001C0A3F5840000001FFF9FEB7028DFC002A093E1B40000001FFBFFC650889F000184500003F994000000100A2FF3600A3FF8065B3352240000001F52AF1243FD5C00028743AE940000001FED7F7BA0E9AF0005FE63D3C40000001FDCBFB003310E000189645314000000102FBFEF524BCC0002FB23FE840000001FFFFFFD10167FF001AE4000038E740000001FFFEFFBB0309FE0016C53E2340000001FF99FC700DD3F8002B863B8B40000001FA5AF88D5127C00014FC3FD640000001FFFFFFAD011DFF0025B600003F5240000001FFF9FEAB02C3FC0027473A8440000001FF39F6EB0914F00057F33FDA40000001FFFFFFB6011DFE0015293F7940000001008DFEF6008EFF00431E00000600060006000600200000000001000020000000000100002000000000010000200000000001000000000000009500000000009513100000000100001310000000010000
6E 2B 2B
# delay: 1.0
13:712:ELM# 2E2B2C3E7752260001FFF8FDA401A5FE0031033E77510A0001FFF0FD9C035AFC00188100003E7751330001FFF5FD9D0251FC0018813E7763E20001FFE9FE1005FCFC0018810000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C0004000000040000001000000004000C00040000000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C0004000400040000001000000004000C00040000000375A40000001FF78FCD41424F0001E5A3ED340000001FFD5FDB90912F8001E563FD540000001FFFFFFAB00C7FF0023643FC840000001FFFFFF91018EFE001C0A3F5840000001FFF9FEB7028DFC002A093E1B40000001FFBFFC650889F000184500003F994000000100A2FF3600A3FF8065B3352240000001F52AF1243FD5C00028743AE940000001FED7F7BA0E9AF0005FE63D3C40000001FDCBFB003310E000189645314000000102FBFEF524BCC0002FB23FE840000001FFFFFFD10167FF001AE4000038E740000001FFFEFFBB0309FE0016C53E2340000001FF99FC700DD3F8002B863B8B40000001FA5AF88D5127C00014FC3FD640000001FFFFFFAD011DFF0025B600003F5240000001FFF9FEAB02C3FC0027473A8440000001FF39F6EB0914F00057F33FDA40000001FFFFFFB6011DFE0015293F7940000001008DFEF6008EFF00431E00000600060006000600200000000001000020000000000100002000000000010000200000000001000000000000009500000000009513100000000100001310000000010000
6E 2B 2C
# delay: 1.0
13:712:ELM# 2E2B43FAE1C8AF967D644B321900
6E 2B 43
# delay: 1.0
13:712:ELM# 2E2B44FAB47D553C28190F080403
6E 2B 44
# delay: 1.0
13:712:ELM#
13:712:ELM# # VIN programming !!!check the command!!!
13:712:ELM# #2EF1905646314A5A315530363436343635323836
13:712:ELM# 3B815646314A5A3155303634363436353238361A6E
7B 81
# delay: 1.0
13:712:ELM#
13:712:ELM# # reset ecu or disconnect the battary!!!check the command!!!
13:712:ELM# 1101
51 01
# delay: 1.0
13:712:ELM#
13:712:ELM# exit
****************************************
*       RESETTING ELM
****************************************
*
*       ERRORS STATISTIC
*
* error_frame      =  0
* error_bufferfull =  0
* error_question   =  1
* error_nodata     =  0
* error_timeout    =  0
* error_rx         =  0
* error_can        =  0
*
*       RESPONSE TIME (Average)
*
* response_time    =  0.102
*
****************************************

**Reply by cardosoparis** [reply]
Hello,

Could you please generate the PyRen ACF autoconfiguration macro for my vehicle?

VIN: VF1RFC00260544918
Platform: XFC
ECU family: 76 – RadNav
Factory software reference: 281154372R
Factory configuration reference: 283C21088R

The ECU currently identifies as:
281152288R
Vdiag 0D
RadNav 3.X v3.3
8.7" portrait display, Bose, 4Control, Easy Park, CarPlay and Multi-Sense.

The PyRen Telegram bot does not respond, and acf.py reports:
“Configuration database not found”.

I only need the family 76 RadNav macro/configuration.
Thank you.

**Comment by mihirat33** [comment]
At speed 38400 insted of General Programming Failure after phone accustic error is Service is not supported in active session

**Reply by shrlnm** [reply]
@mihirat33 Please share the elm log and a dump given in ddt mode

**Reply by saudouin67** [reply]
@shrlnm please I need you so so much 
<img width="1080" height="2340" alt="Screenshot_20260710_095521_Samsung Browser" src="https://github.com/user-attachments/assets/22491398-21ae-4611-86ef-8d09329c8c21" />
I did the exact same thing and u seems to know how ton fix

**Comment by mihirat33** [comment]
[1704965573_R2_08_v3_81_20100413T152018.txt](https://github.com/cedricp/ddt4all/files/13903956/1704965573_R2_08_v3_81_20100413T152018.txt)
[elm_log1.txt](https://github.com/cedricp/ddt4all/files/13903989/elm_log1.txt)
elm_log is second half of it because is too big

**Reply by shrlnm** [reply]
@mihirat33 everything  looks good. Have you tried to reset configuration with 1140 command?

**Reply by mihirat33** [reply]
NO

**Reply by mihirat33** [reply]
Actualy I reseted the cobfiguration using DDT at the very begining. My be that was the mistake that ruined the radio. I trayed to rest it again and executed the macro with the config esterday but nothing new. I was wondering if there is somthing with anty teft safety lock. 

(Multimedia system safety lock
The system is protected by a code to prevent theft. When the system is activated, an identification message "vehicle ID" is sent by the UCH to the gateway via the vehicle's multiplex network, which then sends the message to the other system computers. If the “vehicle ID” stored in the system is the same as the ID received, the system unlocks and authorises the normal operation of its functions.)

Vehicle Identifire are diferen from my old radio VID.

**Reply by shrlnm** [reply]
It is a new information that you have Anti-theft lock. I do not know how to reset acode on  RL1
Your current RL acode is  44 05 17 0F

22281A:62 28 1A 44 05 17 0F 
22281B:62 28 1B 00 00 00 00 
22281C:62 28 1C 00 00 00 00 
22281D:62 28 1D 44 05 17 0F 

What is in system frame 69F?  And what is in your old RL1?

**Reply by mihirat33** [reply]
I dont know how to see it  in system frame 69F but in old RL1 is:

22681A:62 68 1A 68 25 64 6F 
22681B:62 68 1B 00 00 00 00 
22681C:62 68 1C 00 00 00 00 
22681D:62 68 1D 68 25 64 6F

**Reply by mihirat33** [reply]
@shrlnm Can I change this acode or not.

**Reply by shrlnm** [reply]
@mihirat33 I have no practical experience  with RL1. On RL2 the acode may be cleared by virginizing but I don't know how to virginize RL1
Have you tried direct write to this PID? 
2E 68 1A 68 25 64 6F

Not sure that it is possible but I have no other idea

**Reply by mihirat33** [reply]
I am trying to instaall R2_08_V3_81 is it RL1.
I have tried with this:
2E281A6825646F
2E281B00000000
2E281C00000000
2E281D6825646F
but did not succeed.
If you know how to virginiz R2_08_V3_81 this also can be decission.

**Comment by kalinavet** [comment]
У вас получилось запустить?у меня такая жепроблема. нашли решение?

**Comment by Renotech1** [comment]
Here u go :
https://www.swisstransfer.com/d/2f0c2802-39a6-49b5-aa7c-bf108f0394cf

Op do 13 aug 2026, 21:18 schreef cardosoparis ***@***.***>:

**Comment by cardosoparis** [comment]
Merci beaucoup 😇😇😇😇

Château de Montigny

Marcos CARDOSO

1680 Route de Nouziere

Dissay - 86130

☎️:
0623031678

📧:
***@***.***
-------- Message d'origine --------De : Renotech1 ***@***.***> Date : 13/08/2026  22:02  (GMT+01:00) À : cedricp/ddt4all ***@***.***> Cc : cardosoparis ***@***.***>, Comment ***@***.***> Objet : Re: [cedricp/ddt4all] Trying to change my radio (Discussion #939) 
Here u go :

https://www.swisstransfer.com/d/2f0c2802-39a6-49b5-aa7c-bf108f0394cf



Op do 13 aug 2026, 21:18 schreef cardosoparis ***@***.***>:
